packages/slice-machine/src/theme.ts

```diff
@@ -7,7 +7,7 @@ const AppTheme = (): Theme =>
     initialColorModeName: "light",
     colors: {
       text: "#25252D",
-      textClear: "#4E4E55",
+      textClear: "#6F6E77",
```

Comment:
Is this change ok for the other components that use the colour?

Reply:
Normally i'd say no but I had benjamin live with me to validate this. 😅 
The colors are pretty similar, the new one is a bit lighter.